lib/views/pages/newsfeed/newsArticle.dart

```diff
@@ -61,88 +60,112 @@ class _NewsArticleState extends State<NewsArticle> {
       String mutation =
           Queries().createComments(widget.post['_id'], commentController.text);
       Map result = await apiFunctions.gqlmutation(mutation);
+      //print("knfkafasnfak");
```

Comment:
why this print statement required ?

Reply:
actually there was some error in posting the comments . Therefore I was testing that the results are send properly or not .
After that I commented it.